### PR TITLE
Resolve "Setup entry detail page"

### LINKED ISSUE #58: Setup entry detail page

### PR DESCRIPTION
**Pull request by ArsalanSavand**
Closes #58

### CODE REVIEW

File: src/app/components/feed/entry/entry.component.ts [hunk 1]
```diff
@@ -16,23 +21,87 @@ export class EntryComponent implements OnInit {
...
+      this.entry.loading = false;
+    });
+  }
+
+  /**
+   * Bypass security and trust the given value to be safe style value (CSS).
+   *
+   * @param url Entry cover image URL
+   */
+  getBackgroundImage(url: string): SafeStyle {
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: Add to `UtilService`

File: src/app/components/feed/entry/entry.component.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+@Component({
+  selector: 'app-entry',
+  templateUrl: './entry.component.html',
+  styleUrls: ['./entry.component.scss'],
+})
+export class EntryComponent implements OnInit {
+
+  /**
+   * Entry ID
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: Avoid comments like this.

Comment (AmirSavand) [reply src/app/components/feed/entry/entry.component.ts]: Explain what it is, why it's there.

File: src/app/components/feed/entry/entry.component.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+   */
+  private entryId: string;
+
+  /**
+   * Entry
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: Remove obvious comments.

File: src/app/components/feed/entry/entry.component.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+   */
+  entry: EntryFeed;
+
+  /**
+   * Comments
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: Remove obvious comments.

File: src/app/components/feed/entry/entry.component.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+  constructor(private sanitizer: DomSanitizer,
+              private activatedRoute: ActivatedRoute,
+              private entryService: EntryService,
+              private feedService: FeedService) {
+  }
+
+  ngOnInit(): void {
+    this.activatedRoute.params.subscribe((params: Params): void => {
+      this.entryId = params.entryId;
+      // Get entry
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: Remove obvious comments.

File: src/app/components/feed/entry/entry.component.ts [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+      this.entryService.getEntry(this.entryId).subscribe((data: EntryFeed): void => {
+        this.entry = data;
+      }, (error: HttpErrorResponseApi): void => {
+        if (error.status === 404) {
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: What's the point of this empty condition?
Add a new property for error handing: `error: boolean` then set to `true` here.

File: src/app/components/feed/entry/entry.component.ts [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+        }
+      });
+      // Get comments
+      this.entryService.getComments(this.entryId).subscribe((comments: ApiResponse<CommentFeed>): void => {
```
Comment (AmirSavand) [inline src/app/components/feed/entry/entry.component.ts]: Remove obvious comments.

File: src/app/components/feed/feed-routing.module.ts [hunk 8]
```diff
@@ -6,8 +6,34 @@ const routes: Routes = [{
   path: '',
   component: FeedComponent,
   data: {
-    title: 'Feed'
-  }
+    title: 'Feed',
+  },
+  children: [{
+    path: 'explore',
+    loadChildren: './reader/reader.module#ReaderModule',
```
Comment (AmirSavand) [inline src/app/components/feed/feed-routing.module.ts]: Remove all `reader` to `feed`.

File: src/app/services/entry/entry.service.ts [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+@Injectable({
+  providedIn: 'root',
+})
+export class EntryService {
+
+  constructor(private http: HttpClient, private apiService: ApiService) {
+  }
+
+  /**
+   * Get entry
```
Comment (AmirSavand) [inline src/app/services/entry/entry.service.ts]: Change to "Load entry data".

File: src/app/services/entry/entry.service.ts [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+   *
+   * @param id Entry ID
+   */
+  getEntry(id: string): Observable<EntryFeed> {
+    return this.http.get<EntryFeed>(`${this.apiService.base.zero}website/entry/${id}/`);
+  }
+
+  /**
+   * Get entry comments
```
Comment (AmirSavand) [inline src/app/services/entry/entry.service.ts]: Change to "Load all entry comments".

File: src/app/components/feed/feed-routing.module.ts (hunk 8 above)
Comment (ArsalanSavand) [reply src/app/components/feed/feed-routing.module.ts]: Resolved on discussion.

**Pull request merged**